argoverse/utils/sim2.py

```diff
@@ -0,0 +1,106 @@
+"""
+Utility for 2d rigid body transformations with scaling.
+
+Refs:
+    http://ethaneade.com/lie_groups.pdf
+    https://github.com/borglab/gtsam/blob/develop/gtsam_unstable/geometry/Similarity3.h
```

Comment:
This link gives me a 404.

Reply:
good catch -- i just updated it with the correct link